cds_ils/migrator/eitems/api.py

```diff
@@ -145,7 +145,11 @@ def process_files_from_legacy():
 
         try:
             for file_dump in document["_migration"]["files"]:
-
+                if (
+                    file_dump.get("format") == ".ps.gz"
```

Comment:
why this format ?

Reply:
Requested in the [issue](https://github.com/CERNDocumentServer/cds-migrator-kit/issues/92):
```
The files with .ps.gz shouldn't be imported at all (in principle, there is the corresponding version in PDF)
```

Reply:
now when I am thinking, it would be just easier not to add the files to ["_migration"]["files"] if the file is .ps.gz or if is it a yellow report, instead of adding them and just skipping in the iteration